application/src/main/java/org/togetherjava/tjbot/commands/moderation/AuditCommand.java

```diff
@@ -139,26 +149,78 @@ private boolean handleChecks(@NotNull Member bot, @NotNull Member author,
         return ModerationUtils.handleHasAuthorRole(ACTION_VERB, hasRequiredRole, author, event);
     }
 
-    private void auditUser(@NotNull User user, @NotNull ISnowflake guild,
-            @NotNull IReplyCallback event) {
-        List<ActionRecord> actions =
-                actionsStore.getActionsByTargetAscending(guild.getIdLong(), user.getIdLong());
+    private @NotNull List<List<ActionRecord>> groupActions(@NotNull List<ActionRecord> actions) {
```

Comment:
I feel like this method name misses an important opportunity to clarify the actual intention. As in, what are you grouping? Why are you grouping? How are you grouping?

My proposal: `groupActionsByPages` or `paginateActions`.